devito/tools/dtypes_lowering.py

```diff
@@ -128,6 +128,11 @@ def dtype_to_mpitype(dtype):
     }[dtype]
 
 
+def dtype_to_mpidtype(dtype):
```

Comment:
What does this function return exactly?

Reply:
up

Reply:
the MPI data type needed by the mpi comm routines such as alltoall

Reply:
memo